lib/rubocop/cop/rails/active_support_on_load.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Use Active Support lazy load hooks to patch Rails core classes, so they are not forcible loaded early.
```

Comment:
Should this say `forcibly loaded early`?

Reply:
Yep! Thanks :)